.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+aliases:
+  - &restore_sbt_cache
+    restore_cache:
+      key: sbt-{{ checksum "/tmp/scala_version" }}-cache
+
+  - &save_sbt_cache
+    save_cache:
+      key: sbt-{{ checksum "/tmp/scala_version" }}-cache-{{ epoch }}
+      paths:
+        - "~/.ivy2/cache"
+        - "~/.sbt"
+        - "~/cache/coursier"
+
+  - &build_steps
+    - checkout
+    - run: echo "${SCALA_VERSION}" > /tmp/scala_version
+    - *restore_sbt_cache
+    - run:
+        name: Executing cibuild
+        command: ./scripts/cibuild
+    - *save_sbt_cache
+
+version: 2
+workflows:
+  version: 2
+  build:
+    jobs:
+      - "openjdk8-scala2.11.12-nodelts"
+      - "openjdk8-scala2.12.8-nodelts"
+
+jobs:
+  "openjdk8-scala2.11.12-nodelts":
+    docker:
+      - image: circleci/openjdk:8-stretch-node
```

Comment:
The OpenJDK images on Docker Hub don't have `wget` or `curl` installed, so the sbt launcher fails.

Conveniently, CircleCI maintains OpenJDK images that have both `curl` and `node` (for ScalaJS tests) installed. 

It looks like they aren't publishing any tags that correspond to a specific Node release, rather, they're just pinning to the latest LTS in the `Dockerfile`:

https://github.com/CircleCI-Public/circleci-dockerfiles/blob/19efec0c4beb698ff2a1cc597612670433e3fcc3/openjdk/images/8u212-jdk-stretch/node/Dockerfile#L21

Reply:
Using these images seems like a good compromise for now. I rather them maintain the images than us.